### DISCUSSION TITLE
how do use extension in parameterized route?

### DISCUSSION THREAD

**Discussion by ralyodio** [oakserver/oak#618] [Q&A]
```

  .get(
    "/searches/history/tags/:tag.rss",
    controller.getHistoryByTagRss.bind(controller)
  )

```

This appears not to work due to the `.rss` in `:tag.rss` -- is there anyway to to fix this?

**Comment by kitsonk** [comment]
oak uses [path-to-regexp](https://github.com/pillarjs/path-to-regexp) to do the matching.

Named parameters have a default regex match that is too "greedy" for what you are trying to accomplish and you will need to do something more "narrow" like identified [here](https://github.com/pillarjs/path-to-regexp#custom-matching-parameters).

**Comment by ralyodio** [comment]
I'm not understanding, how can I add `:tag.rss` so .rss is not included as the parameter?
